lib/pages/home/survey_carousel/survey_carousel_card.dart

```diff
@@ -22,53 +22,81 @@ class SurveyCarouselCard extends StatelessWidget {
+                  _updateIndicators(0),
+                  const SizedBox(height: 30),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Expanded(
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(
+                              surveyUiModel.title,
+                              style: Theme.of(context).textTheme.headline5,
+                              maxLines: 2,
+                              overflow: TextOverflow.ellipsis,
+                            ),
+                            Text(
+                              surveyUiModel.description,
+                              style: Theme.of(context).textTheme.bodyText1,
+                              maxLines: 2,
+                              overflow: TextOverflow.ellipsis,
+                            ),
+                          ],
+                        ),
+                        flex: 3,
                       ),
-                      flex: 3,
-                    ),
-                    Expanded(
-                      child: FloatingActionButton(
-                        child: const Icon(
-                          Icons.chevron_right,
-                          size: 50,
+                      Expanded(
+                        child: FloatingActionButton(
+                          child: const Icon(
+                            Icons.chevron_right,
+                            size: 50,
+                          ),
+                          backgroundColor: Colors.white,
+                          onPressed: () {
+                            // TODO: Add click action
+                          },
                         ),
-                        backgroundColor: Colors.white,
-                        onPressed: () {
-                          // TODO: Add click action
-                        },
+                        flex: 1,
                       ),
-                      flex: 1,
-                    ),
-                  ],
-                )
-              ],
+                    ],
+                  )
+                ],
+              ),
             ),
           ),
         ],
       ),
     );
   }
+
+  Widget _updateIndicators(int currentPage) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      children: _getIndicators(currentPage),
+    );
+  }
+
+  List<Widget> _getIndicators(int currentPage) {
+    final result = <Widget>[];
+    for (int i = 0; i < 3; i++) {
+      result.add(Container(
+        width: 7.0,
+        height: 7.0,
+        margin: EdgeInsets.symmetric(horizontal: 5),
+        decoration: BoxDecoration(
+          shape: BoxShape.circle,
+          color: (currentPage % 3 == i) ? Colors.white : Colors.white24,
+        ),
+      ));
+    }
+    return result;
+  }
```

Comment:
better create a separate widget for Indicator to avoid big SurveyCarouselCard class 😄

Reply:
ok 🤔